src/utils/constants.ts

```diff
@@ -9,6 +9,8 @@ export const FILE_KEYS: { [key in FileKeys]: string; } = {
 
 export const API_PREFIX = '/api';
 
-export const ASSETS_DIR = './assets';
+export const Images_DIR = './images';
+
+export const Audios_DIR = './audios';
 
 export const RPC_WS_URL = 'ws://127.0.0.1:7887/rpc-ws';
```

Comment:
Leave a blank line in the end of the file. I'd recommend you sent this in your VSCode settings. 


![](https://i.stack.imgur.com/bWSmc.png)